### PR TITLE
feat(agw): fix mme config for containerized agw

### LINKED ISSUE #9350: agw containerization : auto generate mme configuration
Currently, the PR #8424 includes mme config files instead of auto generate them.
this issue is to fix this.

### PR DESCRIPTION
**Pull request by ymasmoudi**
Signed-off-by: YOUSSEF EL MASMOUDI <ymasmoudi@fb.com>

## Summary

Delete mme.conf/spg.conf files used add by pr #8424 
Generate these files from template
remove unnecessary S1_INTERFACE_IP/SGI_INTERFACE_IP config

## Test Plan

built and tested with terravm

```
**** TOTAL number of test cases:1
Check if a test case can be started: if state == hybrid.TC_READYTORUN
Run test case (1) on hybrid: Hybrid_40_4G_IMSI_Attach_&_Traffic()
Wait for the test case to finish or for a timeout: hybrid.tcwait(hybrid.TC_READYTORUN,60)
result: True
Verdict(Hybrid_40_4G_IMSI_Attach_&_Traffic) = VERDICT_PASS

**** Finished testcase **************

Testlist verdict = VERDICT_PASS
Closing NG40 logger: Name = hybrid_40_211025_111755-1; ID = 1
 >>>>>>>> close <<<<<<<<<
```

### CODE REVIEW

File: lte/gateway/deploy/roles/agw_docker/templates/env.j2 [hunk 1]
```diff
@@ -22,12 +22,10 @@ CONFIGS_TEMPLATES_PATH=/etc/magma/templates
...
 CERTS_VOLUME=/var/opt/magma/certs
 CONFIGS_OVERRIDE_VOLUME=/var/opt/magma/configs
+CONFIGS_OVERRIDE_TMP_VOLUME=/var/opt/magma/tmp
 CONFIGS_DEFAULT_VOLUME=/etc/magma
 SECRETS_VOLUME=/var/opt/magma/secrets
 
 HOST_DOCKER_INTERNAL={{ ansible_default_ipv4.address }}
 
-SGI_INTERFACE_IP={{ sgi_interface_ip }}
-S1_INTERFACE_IP={{ s1_interface_ip }}
```
Comment (uri200) [inline lte/gateway/deploy/roles/agw_docker/templates/env.j2:31]: are those not needed anymore?
Maybe @pshelar can confirm this

File: lte/gateway/docker/docker-compose.yaml [hunk 2]
```diff
@@ -142,6 +143,12 @@ services:
       MAGMA_PRINT_GRPC_PAYLOAD: 1
```
Comment (uri200) [inline lte/gateway/docker/docker-compose.yaml]: is MAGMA_PRINT_GRPC_PAYLOAD enabled on purpose?

**Review by uri200 — APPROVED** [review]

File: lte/gateway/deploy/roles/agw_docker/templates/env.j2 (hunk 1 above)
Comment (ymasmoudi) [reply lte/gateway/deploy/roles/agw_docker/templates/env.j2:31]: I added those myself temporary as a part of the first agw containerised version
they were not part of the agw initially.

File: lte/gateway/docker/docker-compose.yaml (hunk 2 above)
Comment (ymasmoudi) [reply lte/gateway/docker/docker-compose.yaml]: Let me disable it.

**Review by ssanadhya — APPROVED** [review]

**Pull request merged**